style.css

```diff
@@ -440,14 +476,20 @@ form > textarea {
   }
 
   .container-flex .top-link {
+    text-decoration: none;
+    font-size: 32px;
+    line-height: 44px;
+    font-weight: 600;
+    font-family: 'Roboto', sans-serif;
+    color: #fff;
     display: none;
   }
 
   .anim {
     margin: 40px 0;
   }
 
-  .container-flex button {
+  .container-flex .btnMenu {
     display: block;
     color: #6070ff;
     border: none;
```

Comment:
- Kindly change the background colour of the hamburger from grey to white as per the Figma design.
Figma design:
<img width="292" alt="Screenshot 2022-04-05 at 7 04 22 PM" src="https://user-images.githubusercontent.com/70269507/161797705-fc1bb51c-a958-4e5b-8c62-5d527c57891a.png">
Your design:
<img width="500" alt="Screenshot 2022-04-05 at 7 08 25 PM" src="https://user-images.githubusercontent.com/70269507/161798225-1a3ec865-220b-4332-b922-673d92231e25.png">

Reply:
Addressed ✔